### PR TITLE
feat: add Extract Data guide

### PR DESCRIPTION
**Pull request by djejaquino**
## Description

Introduces Extract Data (OCR) guide

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: docs/guides/process/extract-data.mdx [hunk 1]
```diff
@@ -0,0 +1,326 @@
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import {Alert} from "@site/src/components/shared/Alert";
+import {AuthButtons} from "@site/src/components/docs/AuthButtons";
+import CardImage from "@site/static/img/guides/process/card.png"
+import CodeBlock from '@theme/CodeBlock';
+
+# Extract Data
```
Comment (armsteadj1) [inline docs/guides/process/extract-data.mdx:8]:
```suggestion
# Extract OCR Data
```

I wonder if we should be more specific about what this is doing?

**Review by armsteadj1 — CHANGES_REQUESTED** [review]

File: docs/guides/process/extract-data.mdx (hunk 1 above)
Comment (djejaquino) [reply docs/guides/process/extract-data.mdx:8]: Fair enough

File: docs/guides/process/extract-data.mdx [hunk 2]
```diff
@@ -0,0 +1,326 @@
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import {Alert} from "@site/src/components/shared/Alert";
+import {AuthButtons} from "@site/src/components/docs/AuthButtons";
+import CardImage from "@site/static/img/guides/process/card.png"
+import CodeBlock from '@theme/CodeBlock';
+
+# Extract OCR Data
```
Comment (armsteadj1) [inline docs/guides/process/extract-data.mdx:8]:
```suggestion
# Extract Data
```

**Review by armsteadj1 — CHANGES_REQUESTED** [review]

**Review by armsteadj1 — APPROVED** [review]

**Pull request merged**